test/query/query.test.ts

```diff
@@ -0,0 +1,174 @@
+import {describe} from "mocha";
+
+const path = require("path");
+const wasm_tester = require("circom_tester").wasm;
+const chai = require("chai");
+const assert = chai.assert;
+
+export {};
+
+const EQUALS  = "0"; // = - equals sign
+const LESS    = "1"; // = - less-than sign
+const GREATER    = "2"; // = - greter-than sign
+const IN = "3"; // = - in
+const NOTIN = "4"; // = - notin
+
+describe("Test query",  function() {
```

Comment:
IMHO less lines of code if use tests.forEach could be better here

Reply:
Yes, but almost all groups have two tests, rewriting them makes just a small Improvement